service_geo_location/test/controllers/GeoLocationControllerIntegrationSpec.scala

```diff
@@ -75,7 +84,10 @@ class GeoLocationControllerIntegrationSpec extends MusitSpecWithServerPerSuite {
         (jsArr.head \ "place").as[String] mustBe "OSLO"
         (jsArr.head \ "zip").as[String] mustBe "0192"
       }
+
+      "return musitHttpError if service not available" in {
+        // todo
```

Comment:
But, you can actually just remove this empty test. The test scenario in the `GeoLocationServiceSpec` should be enough.